src/Microsoft.AspNet.Mvc.Core/ModelBinding/Metadata/DefaultModelMetadata.cs

```diff
@@ -528,5 +529,35 @@ public override IReadOnlyList<object> ValidatorMetadata
                 return _details.PropertySetter;
             }
         }
+
+        public override ValidationErrorMessages ValidationErrorMessages
+        {
+            get
+            {
+                if (_validationErrorMessages == null)
+                {
+                    _validationErrorMessages = new DefaultErrorMessages(ValidationMetadata.ValidationErrorMetadata);
```

Comment:
I'm somewhat ambivalent about creating this read-only wrapper. Could instead change `ValidationErrorMetadata` to `DefaultErrorMetadata` and have it subclass `ValidationErrorMessages` but provide setters. Thoughts?